Source/Core/Core/RmObjEngine.cpp

```diff
@@ -0,0 +1,173 @@
+// Copyright 2013 Dolphin Emulator Project
+// Licensed under GPLv2
+// Refer to the license.txt file included.
+
+
+// RmObjEngine
+// Supports the removal of objects/effects from the rendering loop
+
+#include "Core/ConfigManager.h"
+#include "Core/Core.h"
+#include "Core/RmObjEngine.h"
+
+namespace RmObjEngine
+{
+
+	const char *RmObjTypeStrings[] =
+	{
+		"8bits",
+		"16bits",
+		"24bits",
+		"32bits",
+		"40bits",
+		"48bits",
+		"56bits",
+		"64bits",
+		"72bits",
+		"80bits",
+		"88bits",
+		"96bits",
+		"104bits",
+		"112bits",
+		"120bits",
+		"128bits"
+	};
+
+	static std::vector<RmObj> rmObjCodes;
+
+	void LoadRmObjSection(const std::string& section, std::vector<RmObj>& rmobjects, IniFile& globalIni, IniFile& localIni)
+	{
+		// Load the name of all enabled rmobjects
+		std::string enabledSectionName = section + "_Enabled";
+		std::vector<std::string> enabledLines;
+		std::set<std::string> enabledNames;
+		localIni.GetLines(enabledSectionName, &enabledLines);
+		for (const std::string& line : enabledLines)
+		{
+			if (line.size() != 0 && line[0] == '$')
+			{
+				std::string name = line.substr(1, line.size() - 1);
+				enabledNames.insert(name);
+			}
+		}
+
+		const IniFile* inis[2] = { &globalIni, &localIni };
+
+		for (const IniFile* ini : inis)
+		{
+			std::vector<std::string> lines;
+			RmObj currentRmObj;
+			ini->GetLines(section, &lines);
+
+			for (std::string& line : lines)
+			{
+				if (line.size() == 0)
+					continue;
+
+				if (line[0] == '$')
+				{
+					// Take care of the previous code
+					if (currentRmObj.name.size())
+					{
+						rmobjects.push_back(currentRmObj);
+					}
+					currentRmObj.entries.clear();
+
+					// Set active and name
+					currentRmObj.name = line.substr(1, line.size() - 1);
+					currentRmObj.active = enabledNames.find(currentRmObj.name) != enabledNames.end();
+					currentRmObj.user_defined = (ini == &localIni);
+				}
+				else
+				{
+					std::string::size_type loc = line.find_first_of('=', 0);
```

Comment:
`0` is not necessary. If you're trying to find the first occurrence of "=" in the text line, you should use the regular `find` function. find_first_of finds the first occurrence of any of the characters in the first argument. Since there's only one character, there's no need to use it.